### DISCUSSION TITLE
Cannot run production build after updating to march release

### DISCUSSION THREAD

**Discussion by oanylund** [adonisjs/core#3683] [Help] [answered]
Hi, after updating to march 2022 release I get the following error after building and then trying run the production build isolated:

Cannot find module '@japa/preset-adonis/TestsProvider'

It was added  as a provider in the `.adonisrc.json` file after running `node ace configure tests`. 
It is added as a devdependency in `package.json`. Is this a bug or is there something wrong?

To reproduce: 
- Run the march release
- Build
- Copy build folder out from the project folder so it does not have the parent node_modules folder with the dev dependencies
- Run `node server.js` in the copied build folder

**Comment by RomainLanz** [comment, marked as answer]
Hey @oanylund! 👋🏻 

Please, move the `@japa/preset-adonis/TestsProvider` in your `.adonisjsrc.json` file to the `testProviders` key.

```json
{
  "//": "..."
  "aceProviders": ["@adonisjs/repl"],
  "testProviders": ["@japa/preset-adonis/TestsProvider"],
  "//": "..."
}
```

**Reply by oanylund** [reply]
Thank you, that did it! 

It was the `configure tests` command that added it to providers, but as you already know this it is probably already a fix on the way. or should I make an issue?

**Reply by thetutlage** [reply]
It has already been fixed in this commit. https://github.com/adonisjs/assembler/commit/5f2f65bbb6d31ab815725f8c26bc8379c0f86e0b
